mindmap/mindmap.py

```diff
@@ -287,6 +287,27 @@ def student_view(self, _context=None) -> Fragment:
 
         return frag
 
+    def author_view(self, _context=None) -> Fragment:
```

Comment:
Adding the `author_view` seems great, I only have one suggestion: From the student preview in Studio we should disable the ability to edit the mind map, you could only edit it from the edit modal (student_view).

Reply:
You're right! Thanks

Reply:
Done! Thanks again